uid/src/main/java/com/tomtom/kotlin/uid/Uid.kt

```diff
@@ -44,10 +44,10 @@ class Uid<T> {
      * - Dashes at positions 8, 13, 18, 23 (base 0).
      * - Characters 0-9 and a-f (lowercase only).
      *
-     * if this form is used, the creation of the [Uid] is very fast. If an alternative format
+     * if this format is used, the creation of the [Uid] is very fast. If an alternative format
```

Comment:
```suggestion
     * If this format is used, the creation of the [Uid] is very fast. If an alternative format
```
Third time is the charm? ;)

Reply:
Done...